ipa-core/src/net/test.rs

```diff
@@ -537,42 +608,157 @@ impl TestServerBuilder {
         self
     }
 
-    pub async fn build(self) -> TestServer {
-        let identities =
-            ClientIdentities::new(self.disable_https, ShardedHelperIdentity::ONE_FIRST);
-        let mut test_config = TestConfig::builder()
+    fn test_config(&self) -> TestConfig {
```

Comment:
do we need this function?

I would remove it and have TestServerBuilder just call
```
pub async fn build(self) -> TestServer<Shard> {
        let test_config = TestConfig::builder().build();

        let transport = ...
```

Reply:
yes, this function is used to construct `TestServerBuilder<Shard>` and `TestServerBuilder<Helper>`

Reply:
Sure, it's used on a couple of places... but why not just call `TestConfig::builder().build()` instead  of `self.test_config()`.

I don't see the benefit of test_config method. It just hides the possibility of configuring things.